editors/vscode/package.json

```diff
@@ -185,8 +185,10 @@
     ],
     "scripts": {
         "vscode:prepublish": "npm run build:wasm_lsp-release && npm run compile && shx echo \"GPL-3.0-only OR LicenseRef-Slint-commercial\" > LICENSE.txt",
-        "build:wasm_lsp": "env-var wasm-pack build --dev     --target web ../../tools/lsp --out-dir {{npm_config_local_prefix}}/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
-        "build:wasm_lsp-release": "env-var wasm-pack build --release --target web ../../tools/lsp --out-dir {{npm_config_local_prefix}}/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
+        "build:lsp": "cross-env SLINT_ENABLE_EXPERIMENTAL_FEATURES=1 cargo build -p slint-lsp",
```

Comment:
We can think of a better way to do that.
I think you can set env variable in a build script.
https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env

Reply:
> I did not want to make the LSP force the experimental features at this point for everything. So I went about this piecemeal conversion.

This is internal only, the  rustc-env will only set the env variable when **building**  the LSP.  So this will not enable the experimental feature for our users.  
And we can use experimental feature in our LSP because we use `=1.2.0` in our Cargo.toml file so we control the use of it.  In other words, it is fine to use internal features from the LSP.